openwisp_notifications/tests/test_views.py

```diff
@@ -136,3 +145,20 @@ def test_warning_on_admin_login(self):
         self.assertIn('verify your email address', message)
         expected_url = reverse('notifications:resend_verification_email')
         self.assertIn(expected_url, message)
+
+    def test_warning_disabled(self):
```

Comment:
Let's make the test method name more obvious, eg:
```suggestion
    def test_email_notifications_disabled_no_warning(self):
```